### PR TITLE
Added docs for saved credit card UI

### PR DESCRIPTION
**Pull request by jugarrit**
@mspensieri 
@dannytranlx 
@Mario54 
@lemieux

### CODE REVIEW

File: source/stylesheets/custom.scss [hunk 1]
```diff
@@ -548,7 +548,7 @@ beta:after {
 }
 
 .half-width-img img {
-  max-width: 50%;
+  max-width: 49%;
```
Comment (Mario54) [inline source/stylesheets/custom.scss]: Why?

**Comment by Mario54** [comment]
:+1:

File: source/includes/integrations/_stripe_saved_intro.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+### Saved Credit Cards
+
+We will automatically have Stripe save credit cards for users who are JWT protected.
```
Comment (mspensieri) [inline source/includes/integrations/_stripe_saved_intro.md]: Kind of an awkward sentence

File: source/stylesheets/custom.scss (hunk 1 above)
Comment (jugarrit) [inline source/stylesheets/custom.scss]: 50% doesn't let two images fit side by side.

File: source/includes/integrations/_stripe_saved_intro.md (hunk 2 above)
Comment (jugarrit) [inline source/includes/integrations/_stripe_saved_intro.md]: Should I just say We will automatically save credit cards for users who are JWT protected?

Comment (mspensieri) [inline source/includes/integrations/_stripe_saved_intro.md]: Sounds good

**Pull request merged**